### ISSUE TITLE
[Wallet - Token management] Update token details view

### ISSUE THREAD

**Issue by noeliaSD** [status-im/status-desktop#12373]
### Description

- Review new layout structure. [Design](https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=18700-275878&mode=design&t=79cnqWSiL3pvCdbl-0)
- Actual Design-https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=21330-200075&mode=design&t=wJbfbZaOAjdO7vwH-0
- Update details view (token properties, network information, contract information...):
   - For [Community tokens](https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=17209%3A273085&mode=design&t=STaY7gWkqHmIBOnp-1).
   - For [main token list](https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=18700%3A276834&mode=design&t=4lSOH3LMx1ky1z4D-1) items.
   
![Screenshot 2023-10-06 at 20 52 50](https://github.com/status-im/status-desktop/assets/97019400/2a85cdfe-6fbc-4767-969c-6b181bfff756)

![Screenshot 2023-10-06 at 20 53 14](https://github.com/status-im/status-desktop/assets/97019400/3495113a-be5d-49a3-9ad7-7a0e491a53b6)

**Comment by noeliaSD** [comment]
@benjthayer is working on some updates for these pages so before starting, check if there is new design for them!

**Comment by noeliaSD** [comment]
This task has lower priority than other tasks from the same EPIC. So potential task to be moved to the next iteration.

**Comment by noeliaSD** [comment]
@Khushboo-dev-cpp is this the task you mentioned in the channel that maybe was missing?

**Comment by Khushboo-dev-cpp** [comment]
> @Khushboo-dev-cpp is this the task you mentioned in the channel that maybe was missing?

Yes! you've already created it! thats awesome

**Comment by noeliaSD** [comment]
Hey `UI team`. This is a potential task to be picked up without need of a lot of `Tokens Management` context.
cc: @caybro, @alexandraB99, @alexjba, @micieslak

**Comment by alaibe** [comment]
@benjthayer data for erc20 won't be available in blockchain, any idea on how to get this infor?

**Comment by benjthayer** [comment]
> @benjthayer data for erc20 won't be available in blockchain, any idea on how to get this infor?

Thanks @alaibe could you confirm which data points you are referring to? Thanks

**Comment by Khushboo-dev-cpp** [comment]
> > @benjthayer data for erc20 won't be available in blockchain, any idea on how to get this infor?
> 
> Thanks @alaibe could you confirm which data points you are referring to? Thanks

@benjthayer I think @alaibe was asking about the "minted by" for the ERC20 but now we switched that back to "website" so I guess its all clear now 👍

**Comment by benjthayer** [comment]
> > > @benjthayer data for erc20 won't be available in blockchain, any idea on how to get this infor?
> > 
> > 
> > Thanks @alaibe could you confirm which data points you are referring to? Thanks
> 
> @benjthayer I think @alaibe was asking about the "minted by" for the ERC20 but now we switched that back to "website" so I guess its all clear now 👍

Ah ok! Thanks for the clarification @Khushboo-dev-cpp ! :)
